lib/animas/animasSimulator.js

```diff
@@ -204,7 +206,7 @@ exports.make = function(config){
       if(currSMBG != null && currSMBG.value === event.value) {
         console.log('Duplicate SMBG value (',event.value,')',currSMBG.subType,currSMBG.time,'/',event.subType,event.time);
         var duration = Date.parse(event.time) - Date.parse(currSMBG.time);
-        if ((duration < (15 * sundial.MIN_TO_MSEC)) && (event.subType === 'manual')) {
+        if ((duration < (15 * sundial.MIN_TO_MSEC)) && (event.subType === 'manual') && (currSMBG.subType === 'linked')) {
```

Comment:
It refers to a linked meter, i.e., value was transferred directly from the meter to the pump: http://developer.tidepool.io/data-model/device-data/types/smbg.html#subtype

For Animas pumps, this is only supported by Ping pump when used with Ping meter.